boards/nucleo-f401/Makefile.features

```diff
@@ -0,0 +1,7 @@
+FEATURES_PROVIDED += cpp
+FEATURES_PROVIDED += periph_uart
+FEATURES_PROVIDED += periph_gpio
+FEATURES_PROVIDED += periph_spi
+FEATURES_PROVIDED += periph_cpuid
```

Comment:
Please remove `periph_spi` and `periph_cpuid` as these are not provided by your port.